.github/workflows/cli-rc.yml

```diff
@@ -1,6 +1,10 @@
 name: CLI - Cut Release Candidate
 
-on: push
+on:
```

Comment:
Why is this change needed?

Reply:
Part of this work is to make an update to the GitHub actions to publish CLI RCs on pull requests